cli/cli_state.ml

```diff
@@ -13,7 +13,7 @@ let next_display_mode = function
 type notify_v =
   | Disconnected
   | Connected
-  | Notifications
+  | Notification of Xjid.t
```

Comment:
the `jid.t` is not used -- does your setup still require to have a name there (or did you solve it with the two fds)?